source/quickguides/generate_quick_guides.py

```diff
@@ -31,14 +39,17 @@
 
 # Important: Do not have more than one `#expand` operation per line. The code will only expand the first one.
 def generate_quick_guides():
-    machine_names = ['acme1', 'compy', 'cori']
-    git_top_level = subprocess.check_output('git rev-parse --show-toplevel'.split()).strip().decode('utf-8')
+    machine_names = EXPANSIONS.keys()
```

Comment:
I noticed that there is redundancy with the `machine_names` variable and the `EXPANSIONS` dict's keys.

Also, I ran autopep8 so it did some formatting

Reply:
Still probably makes sense to keep it labeled with the more helpful `machine_names` name.

I mentioned in another comment, but we should try to find a consistent formatting, so pull requests don't get overwhelmed by formatting changes.

Reply:
By redundancy, I mean declaring the array of strings and not using the keys from the dict. I agree about using `machine_names` as the var name still.